README.md

```diff
@@ -51,12 +53,23 @@ Alternatively, you can pre-compile the eBPF program, and provide it to the `trac
 2. `make bpf DOCKER=1` to build in a Docker container which includes all development tooling.
 3. There is also a handy `make all` (and the `make all DOCKER=1` variant) which builds both the executable and the eBPF program.
 
-Once you have the eBPF program artifact, you can provide it to Tracee in any of the locations mentioned above. In this case, the full Docker image can be replaced by the lighter-weight `aquasec/tracee:slim` image (this is unavilable at the moment and will be added to the next release). This image cannot build the eBPF program on its own, and is meant to be used when you already compiled the eBPF program beforehand, for example by adding the following mount to the docker run command: `-v /path/to/tracee.bpf.1_2_3.4_5_6.o:/tmp/tracee/tracee.bpf.1_2_3.4_5_6.o`.
+Once you have the eBPF program artifact, you can provide it to Tracee in any of the locations mentioned above. In this case, the full Docker image can be replaced by the lighter-weight `aquasec/tracee:slim` image. This image cannot build the eBPF program on its own, and is meant to be used when you have already compiled the eBPF program beforehand.
+
+#### Running in container
```

Comment:
in **a** container (or from a container?)

Reply:
mm I'm not sure that's necessary. wouldn't you say "running in linux" for example?